spec/controllers/admin/email_templates_controller_spec.rb

```diff
@@ -0,0 +1,215 @@
+require 'rails_helper'
+
+describe Admin::EmailTemplatesController do
+
+  it "is a subclass of AdminController" do
```

Comment:
I know we use this in a bunch of places in our code base but I'm personally not a fan of it. I would rather we write a simple spec for an anon user to verify that the access is invalid.